### PR TITLE
Remove babel, update dev deps and run audit fix

### PR DESCRIPTION
**Pull request by DonJayamanne**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ module.exports = {
         es6: true,
         node: true
     },
-    extends: ['prettier', 'prettier/@typescript-eslint'],
+    extends: ['prettier'],
```
Comment (DonJayamanne) [inline .eslintrc.js:7]: See https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md#version-800-2021-02-21

File: src/test/datascience/jupyterServer.node.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@
 /** DO NOT USE VSCODE in this file. It's loaded outside of an extension */
 
 import * as crypto from 'crypto';
-import getFreePort from 'get-port';
+import getPort from 'get-port';
```
Comment (DonJayamanne) [inline src/test/datascience/jupyterServer.node.ts:10]: Changes from new version of get-port

File: src/platform/common/variables/envVarsService.unit.test.ts [hunk 3]
```diff
@@ -629,11 +631,14 @@ HAM=$ ... $$ \n\
...
                 // eslint-disable-next-line no-multi-str
-                const vars = parseEnvFile('\
+                const vars = parseEnvFile(
+                    '\
 PYTHONPATH=${REPO}/foo:${REPO}/bar \n\
-                ', {
-                    REPO: '/home/user/git/foobar'
-                });
+                ',
+                    {
```
Comment (DonJayamanne) [inline src/platform/common/variables/envVarsService.unit.test.ts:638]: Changes from running new version of prettier against the code

File: src/kernels/raw/finder/contributedLocalPythonEnvFinder.old.node.unit.test.ts [hunk 4]
```diff
@@ -147,7 +147,7 @@ suite('Contributed Python Kernel Finder', () => {
     });
     test('Status is discovering if Python extension starts refreshing interpreters', async () => {
         when(pythonKernelFinder.kernels).thenReturn([]);
-        const statuses: typeof finder.status[] = [];
+        const statuses: (typeof finder.status)[] = [];
```
Comment (DonJayamanne) [inline src/kernels/raw/finder/contributedLocalPythonEnvFinder.old.node.unit.test.ts:150]: Changes from running new version of prettier against the code

**Review by roblourens — APPROVED** [review]

**Pull request merged**